templates/myfeatures.html

```diff
@@ -0,0 +1,63 @@
+{% extends "_base.html" %}
+
+{% block css %}
+{% endblock %}
+
+{% block subheader %}
+<div id="subheader">
+  <h2>My features</h2>
+</div>
+{% endblock %}
+
+{% block content %}
+<style>
+  ul li {
+      background: white;
+      border: 1px solid gray;
+      margin: 16px;
+      padding: 16px;
+  }
+</style>
+
+<chromedash-accordion
+  id="features_pending_my_approval"
+  title="Features pending my approval"
+  opened>
+
+  <chromedash-feature-table
+    query="pending-approval-by:me"
+    rows="10" columns="approvals">
```

Comment:
I had assumed that the Google HTML style guide said to always use quotes, but when I checked it I found that it does not actually say that.   The value gets set to a number in the custom element regardless of whether we use quotes or not.  Removing the quotes does seem a little cleaner and clarifies the expectation that it would always be a number.  Thanks.